Code/GraphMol/FileParsers/CDXMLParser.cpp

```diff
@@ -116,6 +117,25 @@ std::vector<T> to_vec(const std::string &s) {
   return n;
 }
 
+void scaleBonds(const ROMol &mol, Conformer &conf, double targetBondLength, double bondLength) {
+  double avg_bond_length = 0.0;
+  if(bondLength == -1) {
+    // If we don't have a bond length for any reason, just scale the avgerage bond length
+    for(auto &bond: mol.bonds()) {
+      avg_bond_length += MolTransforms::getBondLength(conf, bond->getBeginAtomIdx(), bond->getEndAtomIdx());
```

Comment:
You don't need this complication (or the include of `MolTransforms` above. You can just use points from the conformer:
```suggestion
      avg_bond_length += (conf.getAtomPos(bond->getBeginAtomIdx()) - conf.getAtomPos(bond->getEndAtomIdx()).length();
```
(that wasn't tested, so it might have a typo or three)